HerPortal/appsettings.json

```diff
@@ -10,7 +10,11 @@
     },
     
     "GovUkNotify": {
-        "BaseUrl": "TODO get real base URL"
+        "BaseUrl": "TODO get real base URL",
+        "ReferralReminderTemplate": {
+            "Id": "383e2ca6-e4f6-46d4-8ab6-00d724a59721",
```

Comment:
Feels odd that we've configured a (presumably real) reminder template id, but not base url?

I assume the ID is only linked to a specific base url...